### PR TITLE
[ICDS Dashboard]: Data pull: Andaman & Nicobar AWC GPS locations for Daily Feeding

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY
https://app.asana.com/0/1183228534052225/1184621296208521/f

### CODE REVIEW

File: custom/icds_reports/utils/data_pull.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        form_location_lat,
+        form_location_long,
+        rank() OVER (
+            PARTITION BY awc_id
+            ORDER BY pse_date DESC
+            )
+
+    FROM tmp_daily_attendance
+
+CREATE TABLE tmp_awc_location_launched AS
```
Comment (Rohit25negi) [inline custom/icds_reports/utils/data_pull.sql:22]: why do you need to create temp table for this?

File: custom/icds_reports/utils/data_pull.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
+CREATE TABLE tmp_daily_attendance AS
+    SELECT
+        awc_id,
+        pse_date,
+        form_location_lat,
+        form_location_long
+        FROM daily_attendance_view WHERE state_id='7fb6f3fe5e7540e7be63c848c28c97ed' AND image_name IS NOT NULL ORDER BY pse_date DESC
```
Comment (Rohit25negi) [inline custom/icds_reports/utils/data_pull.sql:7]: you can put the where clause in the below query only and no need to create this temp table

**Review by Rohit25negi — CHANGES_REQUESTED** [review]

File: custom/icds_reports/utils/data_pull.sql (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/utils/data_pull.sql:22]: it's not possible to join views and having a join over three table increase the cost of query

File: custom/icds_reports/utils/data_pull.sql (hunk 2 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/utils/data_pull.sql:7]: rank function won't work on a view.

**Review by Rohit25negi — APPROVED** [review]
Looks good. Good to verify the result as well before handing it over to GS. also, good to run it on staging before TCL

**Pull request closed without merging**